components/button/button.css

```diff
@@ -0,0 +1,48 @@
+.btn {
+  display: inline-block;
+  text-align: center;
+  vertical-align: middle;
+  cursor: pointer;
+  border: 1px solid transparent;
+  padding: 5px 10px;
```

Comment:
You can declare variables for commonly used sizes like 10px etc, just a suggestion

Reply:
Thanks for suggestion, will change absolute values to variables later